opentelemetry-sdk/src/opentelemetry/sdk/_metrics/point.py

```diff
@@ -74,4 +75,18 @@ class Metric:
     """Contains non-common fields for the given metric"""
 
     def to_json(self) -> str:
-        raise NotImplementedError()
+        return json.dumps(
+            {
+                "attributes": self.attributes if self.attributes else "",
+                "description": self.description if self.description else "",
+                "instrumentation_info": repr(self.instrumentation_info)
+                if self.instrumentation_info
+                else "",
+                "name": self.name,
+                "resource": repr(self.resource.attributes)
```

Comment:
We only print out the attributes of a resource?

Reply:
Yup, this is consistent with `f_span["resource"] = self._format_attributes(self._resource.attributes)` for spans,  and `"resource": repr(self.resource.attributes)` for the log exporter.

Reply:
we *really* should just implement a json serializer for the resource though.

Reply:
Yeah I agree ^. For this Pr though I think this is fine.